src/docs/Updates/Roadmap/index.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Roadmap'
+order: 3
+description: 'Discover what we are working on right now, and how you could contribute to Mozaic'
+---
+
+## Q1 2020 (from 01/01/2020 to 01/04/2020)
+
+#### Create a bêta version of the Mozaic-vue library
+
+- Implement CI / CD and code quality tools
+- Create contributing and coding guidelines
+- Create and migrate at least all the forms components existing in Mozaic into Mozaic-vue
+- Link Mozaic-vue in the Mozaic website
+- Support contribution from other teams
+
+#### Core team components migration into Mozaic
+
+- Help the core team migrate existing components into mozaic
+  - popin / modal component
+  - visibility utilities (for responive and screen readers)
+  - reset / a css file to reset and define basic global styles
+  - images / videos components
+  - separators / dividers component
+  - wysiwyg scope styles (styles required for wysiwyg output)
+
+#### Support Core team into migrating Mozaic into common socle
+
+- Implement mozaic components as freemarker templates
+- Distribute larger parts of Mozaic through common socle
+
+#### Improve Mozaic UI/UX and information architecture
+
+- Redesign home page
+- Improve User interface for a more polished look
+- Improve documentation about contributting
+
+#### Support Feature teams and design studio to contribute to Mozaic
+
+- Organising contributions
+- Following and supporting contributors to improve Mozaic
+
+---
+
+## Want to help ? Here are things that you can work on with us
+
+### Components
+
+- Dialog component
+- Tabs component
+- Tooltips component
+- Breadcrumb component
+- Spinner / loader component
+- Pagination component
+- Carousel component
+- Card component
+- Product Tile Component
+
+### Layouts
+
+- Overlay Layout
+- Toaster layout
+
+### Guidelines / other
+
+- General accessibility Guidelines
+- General tone and writting guidelines
+- General UX principles guidelines
```

Comment:
Good idea ! But maybe it could be a separate page as the "Need Help" page, we could have a "Want to help?" page, that way, it would be a little more visible.

It's just a suggestion. What do you think about it @tiagoux @caqueste @mohamedMok @nherwegh  ?

Reply:
Maybe, but it is very related to the roadmap, I think. I'm not sure about that. 
I think we should discuss priority for those patterns, maybe not display all patterns, but maybe those that are more ready...

Reply:
we already discussed about that but that would be a real added value to have something like a user based kind of decision for the next pattern we'll add in Mozaic